redfish-core/lib/account_service.hpp

```diff
@@ -807,6 +809,81 @@ inline void
         ldapConfigInterface, "UserNameAttribute",
         std::variant<std::string>(userNameAttribute));
 }
+
+inline void acfGetMessageSetProperties(
```

Comment:
Can we remove set from the name? 
getAcfPropertiesFromMessage  or getAcfProperties
It is what we follow upstream

Reply:
Changed.